subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/ReleaseNeededPlugin.java

```diff
@@ -34,32 +34,21 @@
  * Adds following tasks:
  *
  * <ul>
- *     <li>assertReleaseNeeded - {@link ReleaseNeededTask}
- *      - checks if release is needed and fails the build if not needed. Used in release process.</li>
  *     <li>releaseNeeded - {@link ReleaseNeededTask}
  *     - prints information if the release is needed. Useful for testing.</li>
  * </ul>
  */
 public class ReleaseNeededPlugin implements Plugin<Project> {
 
-    public final static String ASSERT_RELEASE_NEEDED_TASK = "assertReleaseNeeded";
     public final static String RELEASE_NEEDED = "releaseNeeded";
 
     @Override
     public void apply(Project project) {
         final ShipkitConfiguration conf = project.getPlugins().apply(ShipkitConfigurationPlugin.class).getConfiguration();
 
-        //Task that throws an exception when release is not needed is very useful for CI workflows
-        //Travis CI job will stop executing further commands if assertReleaseNeeded fails.
-        //See the example projects how we have set up the 'assertReleaseNeeded' task in CI pipeline.
-        releaseNeededTask(project, ASSERT_RELEASE_NEEDED_TASK, conf)
```

Comment:
For backwards compatibility, please keep the task. I suggest to emit deprecated message when the task is executed, including information how to fix the deprecation.